README.md

```diff
@@ -82,9 +82,9 @@ mc.exe --help
 ```
 
 ## Install from Source
-Source installation is intended only for developers and advanced users. `mc update` command does not support update notifications for source based installations. Please download official releases from https://minio.io/downloads/#minio-client.
+Source installation is intended only for developers and advanced users. `mc update` command does not support update notifications for source based installations. Please download official releases from https://min.io/downloads/#minio-client.
```

Comment:
This link 404's, `downloads` changed to `download`, we either need to change the documentation or add a redirect on the main site

Reply:
Both suggestions are done.